migrations/20210612_initial.sql

```diff
@@ -1,21 +1,29 @@
 CREATE TABLE contact_profiles ( -- remote user profile
   contact_profile_id INTEGER PRIMARY KEY,
   contact_ref TEXT NOT NULL, -- contact name set by remote user (not unique), this name must not contain spaces
-  display_name TEXT NOT NULL DEFAULT '',
+  display_name TEXT NOT NULL,
   properties TEXT NOT NULL DEFAULT '{}' -- JSON with contact profile properties
 );
 
--- the first record (id = 1) is reserved for the first local user
-INSERT INTO contact_profiles (contact_profile_id, contact_ref) VALUES (1, '');
-
-
 CREATE TABLE users (
   user_id INTEGER PRIMARY KEY,
-  contact_profile_id INTEGER NOT NULL UNIQUE REFERENCES contact_profiles -- user's profile
+  contact_id INTEGER NOT NULL UNIQUE REFERENCES contacts ON DELETE CASCADE
+    DEFERRABLE INITIALLY DEFERRED,
+  active_user INTEGER -- 1 for active user
```

Comment:
would like to discuss, possibly offline - why at all store `active_user` - only for state between sessions? Other than that `currentUser` in chat controller should be enough, shouldn't it? `selectUser` routine would just be non optional every session start, and it would remove some complexity